public/css/cabinet/cabinetMyResVacCom.css

```diff
@@ -74,4 +74,7 @@ a.orangColor-cabinet:hover
     font-size: 18px;
     /*font-family: "SegoeUIRegular";*/
 
+}
+.topResText{
+    margin-top: -205px;
```

Comment:
Is it correct?

Reply:
moved already existing value in a separate class. Fix bugs.